sbt/Dockerfile

```diff
@@ -20,13 +20,13 @@ ENV JENKINS_HOME /home/jenkins
 ARG MAJOR_MINOR_VERSION
 ARG BASE_URL=https://sbt.bintray.com/rpm/
 
+ENV JAVA_HOME "/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.191.b12-1.el7_6.x86_64"
+ENV PATH "${JAVA_HOME}/bin:${PATH}"
```

Comment:
@nigelhp I've capture your comment about the fact that this may not be needed in ticket 2400